DESCRIPTION

```diff
@@ -30,3 +30,6 @@ Imports:
     duckdb,
     rlang
 URL: https://cdcgov.github.io/cfa-epinow2-pipeline/
+Depends:
+    R (>= 2.10)
```

Comment:
out of curiosity, why not make this higher? If `|>` is used anywhere, wouldn't it need at least 4.1 (or something around there?)

Reply:
It was auto-generated by `usethis::use_data(...)`. I'm not sure why it chose this as the minimum R version -- the functions docs don't say and R packages 2e's data chapter doesn't say either. I assume there's some hard dependency there, but you're right we probably have a hidden dependency somewhere a bit higher.

Reply:
interesting